rosbag2_transport/include/rosbag2_transport/recorder.hpp

```diff
@@ -80,6 +82,22 @@ class Recorder : public rclcpp::Node
   ROSBAG2_TRANSPORT_PUBLIC
   const rosbag2_cpp::Writer & get_writer_handle();
 
+  /// Pause the recording.
+  ROSBAG2_TRANSPORT_PUBLIC
+  void pause();
+
+  /// Resume recording.
+  ROSBAG2_TRANSPORT_PUBLIC
+  void resume();
+
+  /// Pause if it was recording, continue recording if paused.
+  ROSBAG2_TRANSPORT_PUBLIC
+  void toogle_paused();
```

Comment:
Typo in `toogle_paused()` => `toggle_paused()`

Reply:
Fixed in https://github.com/ros2/rosbag2/pull/905/commits/67545366f2cea9100a514abcee498ddaf3b6ed70.